Microsoft.Toolkit.Uwp.UI.Controls/ImageEx/CachingStrategy.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The type of caching to be applied to <see cref="ImageEx"/>.
+    /// Default is <see cref="Custom"/>
+    /// </summary>
+    public enum CachingStrategy
```

Comment:
Given that this enum will only be used for ImageEx, wouldn't it be better to clarify that in the name with something like _ImageExCachingStrategy_?